dgraph/cmd/alpha/run.go

```diff
@@ -771,6 +780,7 @@ func run() {
 			x.UpdateCorsOrigins(origins)
 			return
 		}
+		atomic.AddUint32(&initDone, 1)
```

Comment:
Alpha does not seem to reach this line of code. As a result, initDone is never reaching the value 2.

Reply:
Yes, there was an infinite loop above this line. It should be fixed now.